monkey/monkey_island/cc/repository/i_credentials_repository.py

```diff
@@ -0,0 +1,75 @@
+from abc import ABC
+from typing import Any, Sequence
+
+from monkey.cc.models import ConfiguredCredentials, StolenCredentials
+
+
+class ICredentialsRepository(ABC):
+    def get_configured_credentials(self) -> Sequence[ConfiguredCredentials]:
+        """
+        Retrieves all credentials that were configured.
```

Comment:
I've been using the imperative mood for docstrings ("Retrieve" instead of "Retrieves"). We should standardize. [PEP 257](https://peps.python.org/pep-0257/) uses imperative:

![image](https://user-images.githubusercontent.com/19957806/177197362-59acee67-8bfe-4c95-bda6-181c58a93ba2.png)

![image](https://user-images.githubusercontent.com/19957806/177197149-8f1ff328-ad09-4c20-826d-481dac9de5d3.png)
![image](https://user-images.githubusercontent.com/19957806/177197170-2aaf668e-1539-4c96-9bc4-6694ab1f2851.png)